aix/README.md

```diff
@@ -0,0 +1,67 @@
+Wazuh
+=====
+
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://wazuh.com/community/join-us-on-slack/)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity monitoring, policy monitoring, rootkit detection, real-time alerting, active response, vulnerability detector, etc.
+
+In this repository, you can find the necessary tools to build a Wazuh package for AIX.
+
+## Building RPM packages
+
+To build an AIX package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository, and generate the `.ppc.rpm` package.
+
+1. Download this repository and go to the rpm directory:
+    ```bash
+    $ curl -L https://github.com/wazuh/wazuh-packages/tarball/master | tar zx 
+    $ cd wazuh-wazuh-packages-*
+    $ cd aix
+    ```
+
+2. Execute the `generate_rpm_package.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
+    ```shellsession
+    #./generate_package.sh -h
```

Comment:
A space is missing between `#`  and `./`.

Reply:
+1